processor/processor.go

```diff
@@ -725,7 +724,9 @@ func (proc *Handle) backendConfigSubscriber(ctx context.Context) {
 			sourceIdDestinationMap = make(map[string][]backendconfig.DestinationT)
 			sourceIdSourceMap      = map[string]backendconfig.SourceT{}
 			destinationIDtoTypeMap = make(map[string]string)
+			eventAuditEnabled      = make(map[string]bool)
 		)
+		proc.config.configSubscriberLock.Lock()
```

Comment:
please revert this change, the lock shall be obtains as before, to minimize contention